utils/common/src/main/java/org/apache/brooklyn/util/ssh/BashCommands.java

```diff
@@ -671,6 +672,15 @@ public static String installJavaLatestOrWarn() {
     }
 
     /**
+     * Adds the PPA for OpenJDK for older JDK versions (7 and lower) required by some software (e.g. JBoss)
+     */
+    public static String addOpenJDKPPK(){
```

Comment:
Fair enough too, let's go for that now but I would like to see a better pattern implemented in the future (if possible)